unreal_modloader/src/background_work.rs

```diff
@@ -323,6 +310,8 @@ where
                         })
                         .collect::<Vec<_>>();
 
+                    println!("Integrating {} mods", mods_to_install.len());
```

Comment:
shouldn't this use the logging system? `info!` thingy

Reply:
that was a debugging thing lol